content/en/blog/2020-10-07-helm-hub-moving-artifact-hub.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Helm Hub Moving To Artifact Hub"
+slug: "helm-hub-moving-artifact-hub"
+authorname: "Matt Farina"
+author: "@mattfarina"
+authorlink: "https://mattfarina.com"
+date: "2020-10-07"
+---
+
+Today, we are happy to announce that the Helm Hub is moving to the [Artifact Hub](https://artifacthub.io/). That means, when you go to the Helm Hub you will be redirected to the Artifact Hub.
+
+## What This Means For You?
```

Comment:
No question mark is needed the way this is phrased. Maybe add a "does" if you'd like to keep it as a question?

Reply:
removed question marks